src/cljs/harja/tiedot/urakkatilanne/kojelauta.cljs

```diff
@@ -31,6 +31,27 @@
                             :urakat nil
                             :urakkavuosi (pvm/vuosi (first (pvm/paivamaaran-hoitokausi (pvm/nyt))))}}))
 
+(defn poikkeusten-yhteenveto
+  [urakat]
+  (let [kaikkien-urakoiden-lkm (count urakat)
+        urakat-joissa-avoimia-laatupoikkeamia (count (filter (fn [rivi]
```

Comment:
Ei merkittävää, mutta tuossa countilla otetaan vain urakoiden lukumäärä, ei itse urakkaa, vaikka parametri niin viittaakin.

Reply:
Joo, näin juuri. Koetin tuossa pitää lyhyenä sitä sanamuotoa.. Mietinpä vielä muutanko.